imgui-app/src/main/java/imgui/app/Window.java

```diff
@@ -35,7 +37,7 @@
     /**
      * Background color of the window.
      */
-    protected final Color colorBg = new Color(.5f, .5f, .5f, 1);
+    protected Color colorBg = new Color(.5f, .5f, .5f, 1);
```

Comment:
Why did you remove the `final` modifier`?

Reply:
You weren’t able to modify it since it’s final. How do you change the background color on a class that implements this?

Reply:
- It's a protected variable, so every implementation can access it.
- There is a public getter for the field, so it could be accessed outside the class.
- `Color` class itself has a bunch of methods to modify its value, like: `Color#set(float, float, float, float)`.

Reply:
Ah ok, I didn’t know about `Color#set(float, float, float, float)`.